frontend/src/metabase/embedding-sdk/theme/dynamic-css-vars-config.ts

```diff
@@ -1,25 +1,23 @@
 import type { DynamicCssVarConfig } from "../types/private/css-variables";
 
 /**
- * These CSS variables are dynamically generated based on the theme.
+ * Custom CSS variables that don't correspond to SDK color mappings.
+ * These are always applied and can't be overridden by user-defined colors.
  */
 export const DYNAMIC_CSS_VARIABLES: DynamicCssVarConfig = {
   "--mb-color-bg-sdk-question-toolbar": {
-    light: { source: "bg-white", darken: 0.04 },
-    dark: { source: "bg-white", lighten: 0.5 },
+    light: { source: "background", darken: 0.04 },
+    dark: { source: "background", lighten: 0.5 },
   },
   "--mb-color-notebook-step-bg": {
-    light: { source: "bg-white", darken: 0.05 },
-    dark: { source: "bg-white", lighten: 0.5 },
+    light: { source: "background", darken: 0.05 },
+    dark: { source: "background", lighten: 0.5 },
   },
   "--mb-color-notebook-step-bg-hover": {
-    light: { source: "bg-white", darken: 0.1 },
-    dark: { source: "bg-white", lighten: 0.4 },
-  },
-  "--mb-color-background-hover": {
-    dark: { source: "bg-white", lighten: 0.5 },
+    light: { source: "background", darken: 0.1 },
+    dark: { source: "background", lighten: 0.4 },
   },
   "--mb-color-bg-error": {
-    dark: { source: "error", alpha: 0.3 },
+    dark: { source: "negative", alpha: 0.3 },
```

Comment:
The `error` color that's used for the `SdkError` is not exposed. We can open a follow-up PR to expose it. For now, I'm deriving from the `negative` color as they are similar.